NPE with {# and Qute inlay hint

Create a Qute template file with just:

`{#`

The Qute inlay inlayhint throws an NPE:

```
[Error - 8:38:07 AM] Request textDocument/inlayHint failed.
  Message: Internal error.
  Code: -32603 
java.util.concurrent.CompletionException: java.lang.NullPointerException: Cannot invoke "String.equals(Object)" because "tagName" is null
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyNow(CompletableFuture.java:687)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:662)
	at java.base/java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:2168)
	at com.redhat.qute.services.QuteInlayHint.getInlayHint(QuteInlayHint.java:59)
	at com.redhat.qute.services.QuteLanguageService.getInlayHint(QuteLanguageService.java:216)
	at com.redhat.qute.ls.template.TemplateFileTextDocumentService.lambda$inlayHint$23(TemplateFileTextDocumentService.java:298)
	at com.redhat.qute.ls.commons.ModelTextDocuments.lambda$computeModelAsyncCompose$1(ModelTextDocuments.java:144)
	at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150)
	at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:483)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.NullPointerException: Cannot invoke "String.equals(Object)" because "tagName" is null
	at com.redhat.qute.project.QuteProject.findUserTag(QuteProject.java:436)
	at com.redhat.qute.services.inlayhint.InlayHintASTVistor.createInlayHintParametersSection(InlayHintASTVistor.java:261)
	at com.redhat.qute.services.inlayhint.InlayHintASTVistor.visit(InlayHintASTVistor.java:172)
	at com.redhat.qute.parser.template.sections.CustomSection.accept0(CustomSection.java:62)
	at com.redhat.qute.parser.template.Node.accept(Node.java:338)
	at com.redhat.qute.parser.template.Node.acceptChildren(Node.java:405)
	at com.redhat.qute.parser.template.Template.accept0(Template.java:312)
	at com.redhat.qute.parser.template.Node.accept(Node.java:338)
	at com.redhat.qute.services.QuteInlayHint.lambda$getInlayHint$0(QuteInlayHint.java:74)
	at java.base/java.util.concurrent.CompletableFuture.uniApplyNow(CompletableFuture.java:684)
	... 13 more


```

--- a/qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/project/QuteProject.java
+++ b/qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/project/QuteProject.java
@@ -213,7 +213,7 @@ public class QuteProject {
 	public String getUri() {
 		return uri;
 	}
-	
+
 	public List<QuteProject> getProjectDependencies() {
 		return projectDependencies;
 	}
@@ -423,6 +423,9 @@ public class QuteProject {
 	 * @return user tag by the given tagName and null otherwise.
 	 */
 	public UserTag findUserTag(String tagName) {
+		if (StringUtils.isEmpty(tagName)) {
+			return null;
+		}
 		// Source tags
 		Collection<UserTag> tags = getSourceUserTags();
 		for (UserTag userTag : tags) {
@@ -1514,6 +1517,5 @@ public class QuteProject {
 		return projectRegistry.getJavadoc(new QuteJavadocParams(typeName, getUri(), javaMemberInfo.getName(),
 				signature, hasMarkdown ? DocumentFormat.Markdown : DocumentFormat.PlainText));
 	}
-	
 
 }

--- a/qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/inlayhint/QuteInlayHintTest.java
+++ b/qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/inlayhint/QuteInlayHintTest.java
@@ -29,7 +29,7 @@ import com.redhat.qute.settings.QuteInlayHintSettings;
  *
  */
 public class QuteInlayHintTest {
-
+	
 	@Test
 	public void aliasForSection() throws Exception {
 		String template = "{@java.util.List<org.acme.Item> items}\r\n" + //
@@ -206,6 +206,12 @@ public class QuteInlayHintTest {
 						ihLabel("Object", "Open `java.lang.Object` Java type.", cd("java.lang.Object"))));
 	}
 
+	@Test
+	public void nullTagName() throws Exception {
+		String template = "{#";
+		testInlayHintFor(template);
+	}
+	
 	private static Command cd(String javaType) {
 		return InlayHintASTVistor.createOpenJavaTypeCommand(javaType, QuteQuickStartProject.PROJECT_URI);
 	}

